21_LRUCacheUnitTest/main.cpp

```diff
@@ -145,6 +167,21 @@ class LRUCacheTestApp final : public nbl::application_templates::MonoSystemMonoL
 		#ifdef _NBL_DEBUG
 			cache2.print(m_logger);
 		#endif
+
+			// Grow test - try growing the cache
+			m_logger->log("Growing test");
```

Comment:
call grow more for testing, and sometimes with a number lower than the previous.
(I think `grow` function returns bool, so check for it)

I know it's easy and you handled it, but you're not writing test for the thing you just wrote, you're writing tests for the junior programmer in 2027 trying to touch and modify the LRUCache.
i.e. make it impossible for anyone to pass this test after fucking with the LRUCache.

for example here if I fuck with you and make the grow funciton always fail, we will see no sign of failure, only different log.

TL;DR I'm nitpicking here because this is an important container, I wouldn't have been an*l about this if it was some other example or even test.